pyfilesync/tests/console.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Testing module console.
+"""
+
+from __future__ import unicode_literals
+import unittest
+import sys
+from io import StringIO
+import pyfilesync
+from pyfilesync.console import main
+
+
+# This is tested class. Can have too many method
+class TestConsoleModule(unittest.TestCase):  # pylint: disable=R0904
+    """ Module testsCase. """
+
+    def setUp(self):
+        """ Setup environment. """
+        self.old_stdout = sys.stdout
+        self.old_stderr = sys.stderr
+        sys.stdout = self.tmp_stdout = StringIO()
+        sys.stderr = StringIO()
+
+    def tearDown(self):
+        """ Teardown environment. """
+        sys.stdout = self.old_stdout
+        sys.stderr = self.old_stderr
+
+    def test_console(self):
+        """ Test that console app work ok. """
+        status = 0
+        try:
+            main(["--help"])
+        except SystemExit as ex:
```

Comment:
Ten exception występuje jak zrobimy `sys.exit()`. Normalnie aplikacja zostanie zamknięta, ale jak złapiemy błąd to możemy sobie sprawdzić kod outputu i czy aplikacja zwraca błędy